docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+---
+Order: 10
+Area: cpp
+TOCTitle: Get started with CMake Tools on Linux
+ContentId: 86543311-5452-4b1f-a44c-03cc3df04c3f
+PageTitle: Get started with CMake Tools on Linux 
+DateApproved: 06/11/2020
+MetaDescription: Get started with the CMake Tools Visual Studio Code extension on Linux 
+---
+# Get started with CMake Tools on Linux
+
+CMake is an open-source, cross-platform tool for building, testing, and packaging software. It uses platform and compiler independent configuration files to generate native build tool files specific to your compiler and platform.
+
+The CMake Tools extension integrates Visual Studio Code and CMake to make it easy to configure, build, and debug your C++ project using CMake.
+
+In this tutorial, you'll use the CMake Tools extension for Visual Studio Code to configure, build, and debug a simple C++ CMake project on Linux. Aside from installing CMake and your compiler, debugger, the steps in this tutorial apply generally to how you'd use CMake on other platforms, like Windows.
+
+If you have any trouble, please file an issue for this tutorial in the [VS Code documentation repository](https://github.com/Microsoft/vscode-docs/issues).
+
+## Prerequisites
+
+To successfully complete this tutorial on Ubuntu, install the following:
+
+1. [Visual Studio Code](/download).
+1. [C++ extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools). You can install the C/C++ extension by searching for 'c++' in the Extensions view (`kb(workbench.view.extensions)`).
+
+    ![C/C++ extension](images/cpp/cpp-extension.png)
+
+1. [CMake Tools extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools). You can install the C/C++ extension by searching for 'cmake tools' in the Extensions view (`kb(workbench.view.extensions)`).
+
+    ![CMake tools extension](images/cpp/cmake-extension.png)
+
+1. You'll also need to install CMake, a compiler, a debugger, and a generator.
+
+### Ensure that CMake is installed
+
+The VS Code CMake Tools extension does its work using CMake installed on your system. For best results, use CMake version 3.17 or greater.
```

Comment:
I was thinking 3.15 was the min version we recommended. 3.17 is currently the latest version, but it will be outdated in the coming months and years, so it might be fine to leave it.

Reply:
Set to 3.15

Reply:
Set to 3.15